src/views/home/components/CopyEditionModal.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <b-modal
-        v-if="edition"
+        v-if="currentEditionExists"
```

Comment:
My comments about currentEditionExists and currentEdition from the previous two files are relevant here, as well.